<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网络调试服务端</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>514</width>
     <height>418</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_1" stretch="6,1,1,0,3">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QTextEdit" name="Main_Edit">
      <property name="tabStopDistance">
       <double>80.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="Telecom_Label">
        <property name="text">
         <string>通信协议</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="Telecom_comboBox">
        <item>
         <property name="text">
          <string>TCP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>UDP</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="IP_address_Label">
        <property name="text">
         <string>服务器IP地址</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="IP_address_comboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="Port_number_Lavel">
        <property name="text">
         <string>端口号</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Port_number_Edit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="Listening_pushButton">
        <property name="text">
         <string>监听</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Stop_listening_pushButton">
        <property name="text">
         <string>停止监听</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Disconnection_pushButton">
        <property name="text">
         <string>断开</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Client_name_comboBox" name="Client_comboBox"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QTextEdit" name="Send_textEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="Send_pushButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>发送</string>
        </property>
        <property name="iconSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Client_name_comboBox</class>
   <extends>QComboBox</extends>
   <header location="global">client_name_combobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
